### PR TITLE
Merge master into experimental/undock

### PR DESCRIPTION
**Pull request by fibann**

### CODE REVIEW

File: Microsoft.MixedReality.WebRTC.sln [hunk 1]
```diff
@@ -30,18 +30,24 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "Test", "Test", "{35C3F3A6-2
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestNetCoreConsole", "examples\TestNetCoreConsole\TestNetCoreConsole.csproj", "{C17D2554-9409-4CC7-8337-E3FBE3CAE415}"
 EndProject
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "TestReceiveAV", "examples\TestReceiveAV\TestReceiveAV.csproj", "{209D1A4C-96F1-4F5E-9987-8C64E7998CC3}"
+EndProject
 Global
 	GlobalSection(SolutionConfigurationPlatforms) = preSolution
+		Debug|Any CPU = Debug|Any CPU
```
Comment (djee-ms) [inline Microsoft.MixedReality.WebRTC.sln]: Remove this Solution config and all related Project configs below, they make no sense and make things very confusing for users regarding what architecture is being built for native projects. This was likely auto-added by VS, and we didn't pay attention.

**Comment by fibann** [comment]
> I won't review everything I'm assuming it's fine to go ahead.

Sorry, should have pointed out notable parts. Conflicts weren't too bad, mostly around GlobalFactory so you might want to have a look at that to see if I have missed something.

File: Microsoft.MixedReality.WebRTC.sln (hunk 1 above)
Comment (fibann) [reply Microsoft.MixedReality.WebRTC.sln]: It seems it was added by the new sample project. Fixed.

**Comment by djee-ms** [comment]
> 
> 
> > I won't review everything I'm assuming it's fine to go ahead.
> 
> Sorry, should have pointed out notable parts. Conflicts weren't too bad, mostly around GlobalFactory so you might want to have a look at that to see if I have missed something.

LGTM, can go ahead and merge.

**Pull request merged**